src/Admin/MoocElementAdmin.php

```diff
@@ -7,25 +7,30 @@
 use AppBundle\Form\AttachmentFileType;
 use AppBundle\Form\AttachmentLinkType;
 use AppBundle\Form\PurifiedTextareaType;
+use Doctrine\ORM\QueryBuilder;
 use Sonata\AdminBundle\Admin\AbstractAdmin;
 use Sonata\AdminBundle\Datagrid\DatagridMapper;
 use Sonata\AdminBundle\Datagrid\ListMapper;
 use Sonata\AdminBundle\Form\FormMapper;
 use Sonata\AdminBundle\Form\Type\CollectionType;
 use Sonata\AdminBundle\Form\Type\ModelListType;
 use Sonata\AdminBundle\Route\RouteCollection;
+use Sonata\DatagridBundle\ProxyQuery\ProxyQueryInterface;
 use Symfony\Component\Form\Extension\Core\Type\IntegerType;
 use Symfony\Component\Form\Extension\Core\Type\TextareaType;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 
 class MoocElementAdmin extends AbstractAdmin
 {
-    protected $datagridValues = [
-        '_page' => 1,
-        '_per_page' => 32,
-        '_sort_order' => 'ASC',
-        '_sort_by' => 'position',
-    ];
+    public function createQuery($context = 'list')
+    {
+        /** @var ProxyQueryInterface|QueryBuilder $proxyQuery */
+        $proxyQuery = parent::createQuery($context);
+        $proxyQuery->addOrderBy('o.chapter', 'ASC');
+        $proxyQuery->addOrderBy('o.position', 'ASC');
```

Comment:
What about using annotation like `@ORM\OrderBy({"position": "ASC"})` instead ? what do you think ?